.github/workflows/codeql.yml

```diff
@@ -16,13 +16,51 @@ jobs:
         with:
           go-version: ^1.23
 
+      - uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ vars.GH_APP_ID }}
+          private-key: ${{ secrets.GH_APP_PRIVATE_KEY }}
```

Comment:
Presumably this won't work with forked PRs, will it. I think we probably just have to salvage residual value of the codeql workflow by removing `pull_request` triggers.

Reply:
@rnorth That's unfortunately correct 😢 

> With the exception of GITHUB_TOKEN, secrets are not passed to the runner when a workflow is triggered from a forked repository.

https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions#using-secrets-in-a-workflow